infra/scripts/test_android_runtime_release.sh

```diff
@@ -0,0 +1,142 @@
+#!/bin/bash
+
+[[ "${BASH_SOURCE[0]}" != "${0}" ]] && echo "Please don't source ${BASH_SOURCE[0]}, execute it" && return
+
+CURRENT_PATH="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+ROOT_PATH="$CURRENT_PATH/../../"
+
+: ${DEVICE:="none"}
+TEST_ARCH="aarch64"
+TEST_OS="android"
+TEST_PLATFORM="$TEST_ARCH-$TEST_OS"
+EXECUTORS=("Linear" "Dataflow" "Parallel")
+BACKENDS=( "acl_neon" "cpu" "acl_cl" )
+ANDROID_WORKDIR="/data/local/tmp/onert_android"
+ANDROID_REPORT_DIR="$ANDROID_WORKDIR/report"
+
+# Model download server setting
+if [[ -z "${MODELFILE_SERVER}" ]]; then
+  echo "[ERROR] Model file server is not set"
+  echo "        Need to download model file for test"
+  exit 1
+else
+  echo "Model Server: ${MODELFILE_SERVER}"
+fi
+
+apt-get update && apt-get install -y curl
```

Comment:
I'm not sure but it better not to run this command because it forces user to get `sudo`. But our docker script doesn't have `curl` installed.
@hseok-oh go as it is or add install command to docker script, what do you say?

Reply:
Agree with @mhs4670go

Reply:
then, I think we need to update `ONE/infra/docker/<UbuntuReleaseCode>/Dockerfile` to install `curl`. is this enough for it?

Reply:
@struss Yes you can post related PR.